OpenOversight/app/static/js/find_officer.js

```diff
@@ -37,9 +37,14 @@ $(document).ready(function() {
             data: {department_id: dept_id}
         }).done(function(ranks) {
             $('input#rank').replaceWith('<select class="form-control" id="rank" name="rank">');
+            var rank_box = $('select#rank')
```

Comment:
Nothing serious, but generally `var` has some strange edge-case behaviors we can avoid using `let` or `const` these days 🙂 

```suggestion
            const rank_box = $('select#rank')
```

Reply:
Happy to change this haha, I was just staying consistent with the rest of the file :sweat_smile: